src/main/java/com/cleanroommc/modularui/api/drawable/IRichTextBuilder.java

```diff
@@ -30,7 +30,16 @@ default T addLine(IDrawable line) {
     }
 
     default T newLine() {
-        return add(IKey.LINE_FEED);
+        return newLine(1);
```

Comment:
Since `newLine()` is called frequently, i dont think its  a good idea to this here. It would be better to solve in the renderer or compiler.